lms/djangoapps/instructor_task/tests/test_base.py

```diff
@@ -383,7 +388,7 @@ def _extract_and_round_numeric_items(dictionary):
         to four decimal places.
         """
         extracted = {}
-        for key, value in dictionary.items():
+        for key, value in list(dictionary.items()):
```

Comment:
convert the iterator to a list because changing the dictionary that we're iterating gives a runtime error.